cdap-docs/integrations/source/partners/cloudera/configuring.rst

```diff
@@ -186,10 +188,69 @@ The following is the generic procedure for Major/Minor version upgrades:
 
 .. _cloudera-release-specific-upgrade-notes:
 
+.. rubric:: Upgrading CDAP 3.0 to 3.1 and Upgrading CDH 5.3 to 5.4
```

Comment:
order should be reversed, ``Upgrading CDH 5.3 to 5.4 and Upgrading CDAP from 3.0 to 3.1``